src/scss/_main.scss

```diff
@@ -419,18 +419,37 @@ nav {
   &.-left {
     justify-content: flex-start;
   }
+
+  &.-right {
+    justify-content: flex-end;
+  }
 }
 
 .build .error {
   color: var(--color-red-light);
 
   font-size: 16px;
+
+  max-width: 80%;
 }
 
 .build .error .message {
   margin-left: 0.2em;
 }
 
+.log-actions {
+  display: flex;
+  justify-content: flex-end;
+  margin-top: 0.2rem;
+  margin-right: 1rem;
+  background: linear-gradient(
```

Comment:
does this need to be a gradient? i think simple background color is fine here?

Reply:
thats actually the flowline that connects step icons on the left side. I've renamed the class to `flowline-left` and adapted it to the new log-actions, removing it from this class ^ Feel free to open another comment if you want but that linear gradient is how I get it to work well with icon padding and overflow-hidden

Reply:
yup, you're right, my fault. i think what happened is that i toggled on/off and thought it was doing the opposite - blocking the line out. nothing to see here :)